pyresample/future/spherical/arc.py

```diff
@@ -58,15 +60,11 @@ def __hash__(self):
 
     def is_on_equator(self):
```

Comment:
Should this be an `on_equator` property? Or `is_on_equator` I guess if we want to keep the verb.